diff --git a/CMakeLists.txt b/CMakeLists.txt
index e176b6a7..929cd452 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -432,6 +432,7 @@ if(CL_API)
 else()
   find_package(Protobuf 3.0.0 REQUIRED)
 endif()
+find_package( absl REQUIRED )

 if(Protobuf_VERSION VERSION_LESS 3.2)
   add_definitions(-DPROTOBUF_SET_BYTES_LIMIT)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 85558737..542065cd 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -140,6 +140,9 @@ target_link_libraries(
          # Link in this static lib, but don't make it a transitive dependency of
          # TestGTIRB, etc
   PRIVATE gtirb_proto
+         absl::log_internal_message
+         absl::log_internal_check_op
+         absl::status
 )
 target_compile_definitions(
   ${PROJECT_NAME} PRIVATE GTIRB_${PROJECT_NAME}_EXPORTS
